### PR TITLE
Footer navbar

### PR DESCRIPTION
**Pull request by danielab93**
#3 I've created a footer-navbar with svgs so that the user can navigate between the different menu-points :)

### CODE REVIEW

File: src/components/FooterNavbar.jsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+              fill=""
+              stroke=""
+              strokeWidth="2"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            />
+          </svg>
+        </NavLink>
+      </NavbarStyled>
+      <Outlet />
```
Comment (anitaschiller) [inline src/components/FooterNavbar.jsx]: Is the Outlet really needed here?

**Review by anitaschiller — CHANGES_REQUESTED** [review]
Hi Daniela, good job! Just one small comment and you are good to go :-).

File: src/components/FooterNavbar.jsx (hunk 1 above)
Comment (danielab93) [reply src/components/FooterNavbar.jsx]: No it's actually not needed so I deleted it. Please review my code again - I'm looking forward to your approval :)

**Review by anitaschiller — APPROVED** [review]
Nice, can be merged from my side ;)

**Pull request merged**